spec/takeaway_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  subject(:takeaway) { described_class.new(menu_klass.new, order_klass.new) }
+
+  let(:menu) {double(:menu, dishes: {'Spring Roll'=>0.99, 'King Prawn'=>2.99})}
+  let(:menu_klass) { double(:menu_klass, :new => menu) }
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)